stanford/roles/cluster/defaults/main.yml

```diff
@@ -21,6 +21,16 @@ CLUSTER_SECURITY_GROUPS:
 
 CLUSTER_NETWORK: private
 CLUSTER_SUBNETS:
+  analytics_api:
+    - zone: 'us-west-1a'
+      cidr: "{{ DEPLOYMENT_CIDR_BASE }}.155.0/24"
+    - zone: 'us-west-1c'
+      cidr: "{{ DEPLOYMENT_CIDR_BASE }}.156.0/24"
+  analytics-jenkins:
```

Comment:
Looking upstream, they don't seem to have a play for a dedicated jenkins box.
I don't think there's much specialized for insights, so can we make it a general purpose jenkins box for the entire deployment?
Similar to how we no longer need a dedicated jump or "director" box, but rather share the deployment's existing jump and "deploy" boxes..